bodhi/util.py

```diff
@@ -440,7 +440,13 @@ def update2html(context, update):
     else:
         title = update['title']
 
-    url = request.route_url('update', id=title)
+    alias = None
+    if hasattr(update, 'alias'):
+        alias = update.alias
+    else:
+        alias = update.get('alias')
```

Comment:
I don't think the `if/else` conditional is necessary, since `alias` will always exist. It defaults to `None`, but all updates get assigned an alias immediately upon submission.